develop/tutorials/articles/240-javascript-module-loaders/20-loading-modules-with-aui-script-tag.markdown

```diff
@@ -0,0 +1,76 @@
+# Loading Modules with AUI Script in @product@ [](id=loading-modules-with-aui-script-in-liferay)
+
+## What is the aui:script Tag?
+
+The `aui:script` tag is a tag for JSPs that allows you to load JavaScript in script tags on the page and ensure certain resources are loaded before executing.
+
+## Using aui:script [](id=using-aui-script)
+
+The `aui:script` tag supports the following options:
+
+- **require**: Requires an AMD module that will be loaded with the [Liferay AMD Module Loader](https://github.com/liferay/liferay-amd-loader#amd-module-loader).
+- **use**: Uses an AlloyUI/YUI module that will be loaded via the YUI loader
+- **position**: The position the script tag is put in the page.  Can be `inline` or `auto`.
+- **sandbox**: If set to true, it will wrap the script tag in an anonymous function.  Also, `$` and `_` will be defined for jQuery and underscore.
+
+## Loading ES2015 and Metal.js Modules [](id=loading-es2015-and-metaljs-modules)
+
+You can use `aui:script` to load your ES2015 and Metal.js modules like this:
```

Comment:
Hey @mwilliams2014, this corresponds to the example below. Simply remove the next line so everything makes sense